src/components/organisms/pupil/OakQuizHint/OakQuizHint.tsx

```diff
@@ -2,35 +2,37 @@ import React, { MouseEventHandler, ReactNode, useState } from "react";
 
 import { OakTooltip } from "@/components/molecules";
 import { OakHintButton } from "@/components/organisms/pupil/OakHintButton";
+import { OakBox } from "@/components/atoms";
 
 export type OakQuizHintProps = {
   /**
    * Some content to give as a hint to answer a question
    */
   hint: ReactNode;
+  id: string;
 };
 
 /**
  * Presents a button which will show a hint when clicked
  */
-export const OakQuizHint = ({ hint }: OakQuizHintProps) => {
+export const OakQuizHint = ({ hint, id }: OakQuizHintProps) => {
```

Comment:
might be worth defaulting the id in any case ?

Reply:
I considered that, but then I'd have to make `id` optional and figured it is better to draw attention to the fact an `id` it is needed (if it ever gets used elsewhere). The other option was to use a generated guid for the id, but I figured I was overthinking it 🙂